legend-engine-protocol-pure/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/mapping/Mapping.java

```diff
@@ -14,20 +14,24 @@
 
 package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.mapping;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.PackageableElement;
 import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.PackageableElementVisitor;
-import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.mapping.mappingTest.MappingTest;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.mapping.mappingTest.MappingTestSuite;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.mapping.mappingTest.MappingTest_Legacy;
 
 import java.util.Collections;
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
why we need this?

Reply:
making it backward compatible , any protocol model should anyways have this no?

Reply:
I don't think we need this Jackson property. Thats only if we have properties coming in from the json that we havent accounted for which we are still accounting for `tests`. I would say could we move the initialization of tests to be null instead of empty list so when we seralize to json it is not present as we move to remove this altogether

Reply:
I believe I agree with @MauricioUyaguari, we should remove the initialization.
Though I would be fine if we want to do that as separate PR as we would need to ensure null checks at multiple places.
Though we definitely not need the annotation